src/microbe_stage/Microbe.cs

```diff
@@ -2108,6 +2110,82 @@ private Transform GetNewPhysicsRotation(Transform transform)
         return new Transform(new Basis(slerped), transform.origin);
     }
 
+    /// <summary>
+    ///   Adds torque to reach the targeted rotation
+    /// </summary>
+    private void ApplyPhysicsRotation(PhysicsDirectBodyState physicsState)
+    {
+        var transform = physicsState.Transform;
+        var target = transform.LookingAt(LookAtPoint, new Vector3(0, 1, 0));
+        if (transform == target)
+            return;
+
+        var strength = Constants.CELL_BASE_THRUST + Constants.CELL_BASE_THRUST * (HexCount/2);
+        var maxVelocity = 5.0f;
+
+        var distance = target.basis.GetEuler().y - transform.basis.GetEuler().y;
+        distance = (float)(distance * (180/Math.PI));
+        if (distance > 180) {
+            distance = -360 - distance;
+        } else if (distance < -180) {
+            distance = 360 + distance;
+        }
+
+        var force = 0.0f;
+        var velocity = physicsState.AngularVelocity.y;
+
+        if (velocity < -maxVelocity) {
+            force = 1;
+        } else if (velocity > maxVelocity) {
+            force = -1;
+        } else {
+            float velocityTo90 = 0.0f;
+            if (velocity != 0)
+                velocityTo90 = ((velocity)/maxVelocity) * 90;
```

Comment:
Why not simply use `float velocityTo90 = velocity / maxVelocity * 90.0f;` (or pi/4 if you go in rad)?